src/test/java/com/bravo/user/controller/PaymentControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package com.bravo.user.controller;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.service.PaymentService;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@SpringBootTest()
+@AutoConfigureMockMvc
+public class PaymentControllerTest {
+    
+    @Autowired
+	private MockMvc mockMvc;
+
+    @MockBean
+	private PaymentService paymentService;
+
+    private List<PaymentDto> payments;
+
+    @BeforeEach
+	public void beforeEach() {
+        log.info("Initializing  payment list");
```

Comment:
I think logs in tests are usually unnecessary as there are methods built into the testing tools to provide more clear output.